src/AuthModule/components/ChangPass/ChangPass.tsx

```diff
@@ -0,0 +1,195 @@
+          },
+        }
+      );
+      console.log(response)
+      handleClose();
+      showSuccessToast("Password has been updated successfully");
+    } catch (error) {
+      if (axios.isAxiosError(error) && error.response) {
+        showErrorToast(error.response.data.message);
+      }
+    } finally {
+      setSpinner(false);
+    }
+  };
+
+  return (
+    <>
+      <div className=" row justify-content-center align-items-center">
+        <div className=" col-md-12">
+          <div className="  rounded-3 px-5 py-4">
+            <small className=" text-white">welcome to PMS</small>
+
+            <h4 className={`${styleChangPass.changeWord} mb-5 `}>
+              {" "}
+              Change Password
+              <div className="color "></div>
+            </h4>
+
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <div className="form-group conInput  mb-3">
+                <label className=" color-text" htmlFor="old ">
+                  Old Password
+                </label>
+                <input
+                  className={`${styleChangPass.styleInput} form-control mt-2 `}
+                  type={showPassword ? "text" : "password"}
+                  placeholder="Enter your Old Password  "
+                  {...register("oldPassword", {
+                    required: "old Password is required ",
+                  })}
+                  id="old"
+                />
+                <i
+                  className={`fa-regular fa-eye${showPassword ? "-slash" : ""}`}
+                  onClick={togglePasswordVisibility}
+                ></i>
+
+                {errors.oldPassword && (
+                  <div className="alert alert-danger  d-inline-block w-100 mt-1">
+                    {errors.oldPassword.message}
+                  </div>
+                )}
+                <div className="border_bottom"></div>
+              </div>
+
+              {/*new password */}
+
+              <div className="form-group conInput  my-4">
+                <label className=" color-text" htmlFor="New ">
+                  New Password
+                </label>
+                <input
+                  className={`${styleChangPass.styleInput} form-control mt-2 `}
+                  placeholder="Enter your New Password"
+                  type={showNewPassword ? "text" : "password"}
+                  {...register("newPassword", {
+                    required: "new Password is required",
+                    pattern: {
+                      value:
+                        /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[a-zA-Z]).{8,}$/,
+                      message:
+                        "Password must contain at least 8 characters, including upper and lowercase letters, and numbers",
+                    },
+                  })}
+                  id="New"
+                />
+                <i
+                  className={`fa-regular fa-eye${
+                    showNewPassword ? "-slash" : ""
```

Comment:
I think we should flip the icons to represent the current state:
eye slash: the password is hidden and you can't see the password (because there's slash over the eyes)
eye: the password is visible
![image](https://github.com/AnasKhaled7/PMS-React/assets/64327685/ddfb281a-ad7d-440c-8d55-149155b88e7e)

We can also look at a real example if we forget such as facebook:
![image](https://github.com/AnasKhaled7/PMS-React/assets/64327685/36f7d94f-6e67-4aeb-952a-05fb9badcb6b)